app/classifier/components/Intervention/Intervention.spec.js

```diff
@@ -32,10 +32,9 @@ describe('Intervention', function () {
     const newlineMsg = intervention.message + "\n"
     expect(wrapper.find(Markdown).first().text()).to.equal(newlineMsg);
   });
-  it('should show an opt-out message', () => {
+  it('should show a study info message', () => {
     const expectedMarkdown = 'I do not want to take part in this messaging [study]' +
-      '(+tab+https://docs.google.com/document/d/1gLyN6Dgff8dOCOC88f47OD6TtFrfSJltsLgJMKkYMso/preview).' +
-      ' Do not show me further messages.'
+      '(+tab+https://docs.google.com/document/d/1gLyN6Dgff8dOCOC88f47OD6TtFrfSJltsLgJMKkYMso/preview).'
```

Comment:
Out of interest, is the concatenation here just because the linter complains about long lines?

Reply:
it was to help me see the content of the markdown on my small screens